jobs/build-arch.Jenkinsfile

```diff
@@ -315,7 +315,8 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
         stage("Kola") {
             def n = 4 // VMs are 2G each and arch builders have approx 32G
             kola(cosaDir: env.WORKSPACE, parallel: n, arch: basearch,
-                 allowUpgradeFail: params.ALLOW_KOLA_UPGRADE_FAILURE)
+                 allowUpgradeFail: params.ALLOW_KOLA_UPGRADE_FAILURE,
+                 skipSecureBoot: pipecfg.hotfix?.skip_secureboot_tests_hack)
```

Comment:
what does `pipecfg.hotfix?.skip_secureboot_tests_hack` evaluate to if that isn't in the config? `null`?